### PR TITLE
Add support for chefkoch.de

### PR DESCRIPTION
**Pull request by MRuy**
Add support for the german chefkoch.de website

Known problem:
Line breaks getting removed from directions

### CODE REVIEW

**Comment by MRuy** [comment]
Known problems:
* Recipes can have more then one ingredients list, they will get merged together
* Directions using <br> instead of paragraphs for line breaks, it will make the text without line breaks

File: lib/domain/annoyingDomains.ts [hunk 1]
```diff
@@ -137,6 +137,40 @@ const getTastyCoData = (html: string): RecipeMetadata => {
...
+        $(element).find('tr td').map((index, element) => {
+            const text = $(element).text().trim();
+            if (index % 2 === 0) {
+                ingredientAmounts.push(text);
+            }
+            else {
+                ingredientNames.push(text);
+            }
+        });
+        ingredientSections.push({
```
Comment (jpbulman) [inline lib/domain/annoyingDomains.ts:159]: This is great! :medal_sports:

**Comment by jpbulman** [comment]
Thanks for the work, this looks great to me! The only minor thing is to just resolve the conflict/alphabetize the domain list (looks like I let one slip in un-alphabetized :facepalm:)

Other than that, it's merge-ready!

**Comment by DillonStreator** [comment]
Fixes #23

**Comment by jpbulman** [comment]
Fantastic, seems to work fine for me now! Merging :partying_face: 

Thanks for the contributions!

**Pull request merged**